### DISCUSSION TITLE
Suggestions: Interlacing and Night-Vision effect

### DISCUSSION THREAD

**Discussion by notdigidani** [ntsc-rs/ntsc-rs#95] [Ideas]
basically what the title says, maybe add it on the "VHS Emulation" section as a switch or something like that

**Comment by dawnch0rus** [comment]
interlacing isn't something that you can really fake. that being said, ntsc-rs does support actual interlaced video. see my comment at https://github.com/valadaptive/ntsc-rs/issues/54#issuecomment-1982072517 for a hack to add real interlacing to a video file.

what does a "night vision effect" entail? i don't think any kind of night vision was part of any ntsc format

**Reply by notdigidani** [reply]
thanks for the interlacing hack

and for the night vision effect, i thought it would be a cool addition to the software. it wasnt part of any ntsc format, but i thought it would be an interesting video effect, i dont know.

**Reply by Degamisu** [reply]
Wait, im pretty sure it does. There is a setting (where the alternating fields setting is, not sure the name) called `interleave` which basically means the output can become interlaced. also, the nightvision effect isn't something you can fake. the camera shoots out infared rays which bounce back to the camera, enbrightening detail without a flash. Usually the output looks like a flash, which isn't possible without some sort of realtime AI depth effect.

**Reply by notdigidani** [reply]
oh yes there is, thanks!

well, its a shame that it isnt possible to fake an nightvision effect. it would be cool tho

anyway, thanks for the help

**Comment by valadaptive** [comment]
ntsc-rs is specifically a VHS effect. Other effects are outside its scope.

**Reply by notdigidani** [reply]
i coudnt agree more. any other VHS effect out there feels crappy, cheap and fake

**Reply by Degamisu** [reply]
Indeed. like.. whatever this is 😆 

![image](https://github.com/valadaptive/ntsc-rs/assets/149022474/8af89ffa-c7bb-4ce6-9d8b-a72c17f0d90a)
